exodus/exodus/wsgi.py

```diff
@@ -7,10 +7,9 @@
 https://docs.djangoproject.com/en/1.11/howto/deployment/wsgi/
 """
 
-import os
-
 from django.core.wsgi import get_wsgi_application
 
+# import os
```

Comment:
Do we really need these two commented lines? What do you think about removing them?

Reply:
You're right, I'll remove them :)